src/main/java/org/traccar/config/Keys.java

```diff
@@ -636,19 +636,28 @@ private Keys() {
             "openid.clientSecret",
             List.of(KeyType.CONFIG));
 
+    /**
+     * OpenID Connect Issuer (Base) URL.
+     * This is used to automatically configure the authorization, token and user info URLs if
+     * they are not provided.
+     */
+    public static final ConfigKey<String> OPENID_ISSUERURL = new StringConfigKey(
```

Comment:
Actually can we update the keys to separate works. Haven't noticed it in the original PR. It should be something like `OPENID_ISSUER_URL` for this one.

Reply:
To confirm, the config file property names are fine, just the internal need changing?

Reply:
Yes, just the Java keys.